gbasis/density.py

```diff
@@ -0,0 +1,44 @@
+"""Density Evaluation."""
+import numpy as np
+
+
+def eval_density(charge_density, cart_cont_orb):
+    """Return the evaluation of the density at different points.
+
+    Parameters
+    ----------
+    charge_density : np.ndarray(L,L,)
+        Charge density matrix.
+    cart_cont_orb : np.ndarray(L,N,)
```

Comment:
You may want to specify that the `cart_cont_orb` can support different dimensions: `{np.ndarray(L), np.ndarray(L, N), np.ndarray(L, N, N, N)}`. 

I'm not sure how to specify that it can handle arbitrary dimension/shapes as long as the first dimension has the correct size. If there is a way, then that's great. If not, then the given cases should be the most common ones.